tests/gnrc_sixlowpan_frag_minfwd/mockup_netif.c

```diff
@@ -38,9 +38,8 @@ void _common_set_up(void)
 {
     expect(_mock_netif != NULL);
     gnrc_ipv6_nib_init();
-    gnrc_netif_acquire(_mock_netif);
     gnrc_ipv6_nib_init_iface(_mock_netif);
-    gnrc_netif_release(_mock_netif);
+    gnrc_ipv6_nib_iface_up(_mock_netif);
```

Comment:
And this one?

Reply:
No, i mean:

https://github.com/RIOT-OS/RIOT/blob/04b7ea59fd6398b98f29b6caf8a8af6f7a7e3010/tests/gnrc_sixlowpan_frag_minfwd/main.c#L311-L312

Is it also required here, or not?

Reply:
Thank you, added